src/Lifecycle.js

```diff
@@ -131,8 +131,8 @@ export function getStoredSessionOwner() {
  *     for a real user. If there is no stored session, return null.
  */
 export function getStoredSessionIsGuest() {
-    const {hsUrl, isUrl, accessToken, userId, deviceId, isGuest} = _getLocalStorageSessionVars();
```

Comment:
Hmm, I guess you also could have removed the unused ones (`deviceId` and `isUrl`) to satisfy the linter...? Anyway, either is fine of course.

Reply:
oh, yes - I think I was getting confused between array and object destructuring.